tuf/src/repository/ephemeral.rs

```diff
@@ -147,20 +143,18 @@ mod test {
             let repo = EphemeralRepository::<Json>::new();
 
             let data: &[u8] = b"like tears in the rain";
-            let target_description =
```

Comment:
What was this meant to represent?

Reply:
`TargetDescription` contains things like the file hashes and any custom fields. A long time ago, we just had a single `Repository` trait, and implementations needed to do all the validation, but we eventually split the trait up into `RepositoryProvider` and `RepositoryStorage`, and added a separate internal `Repository` type that handles validation in one place. We just forgot to clean up these arguments when we made the split.